app/tests/conftest.py

```diff
@@ -43,8 +43,9 @@ def fresh_db():
     try:
         yield {"dbPath": chroma_db_path, "collectionName": collection_name}
     finally:
-        if os.path.exists(chroma_db_path):
-            shutil.rmtree(chroma_db_path)
+        chroma_client = chromadb.PersistentClient(path='chromadb_store_test')
+        chroma_client.delete_collection(name=collection_name)
+
         db_conn = psycopg2.connect(
```

Comment:
Outside of the scope of the pr but having a utility function to run the postgress commands would make it easier to read.